packages/office-ui-fabric-react/src/components/Breadcrumb/Breadcrumb.tsx

```diff
@@ -84,13 +84,21 @@ export class Breadcrumb extends BaseComponent<IBreadcrumbProps, IBreadcrumbState
             ) : (null) }
             { renderedItems.map(
               (item, index) => (
-                <li className='ms-Breadcrumb-listItem' key={ item.key || String(index) } ref={ item.key || String(index) } >
-                  <a className='ms-Breadcrumb-itemLink'
-                    onClick={ item.onClick ? this._onBreadcrumbClicked.bind(this, item) : null }
-                    href={ item.href }
-                    role={ item.onClick ? 'button' : 'link' }>
-                    { item.text }
-                  </a>
+                <li className='ms-Breadcrumb-listItem' key={ item.key || String(index) } ref={ item.key || String(index) }>
+                  { (() => {
```

Comment:
why do you use that IIFE here? might be good to move it to a class method, something like `_renderItem` and call that